src/en/page-templates/basic/code.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Basic page code
+layout: 'layouts/page-templates-overview.njk'
+eleventyNavigation:
+  key: basicPageCodeEN
+  locale: en
+  parent: basicPageTemplateEN
+  state: published
+translationKey: 'basicPageCode'
+date: "git Last Modified"
+---
+
+# Copy this code
```

Comment:
Can we change the `h1` text on this page to something that mentions the template name? That way when we have more templates on the site we don't have multiple pages with the same `h1`.

Reply:
Yes, I thought the same thing. @melaniebmn and I worked on some text last week and I updated in the docs [EN](https://docs.google.com/document/d/1D0gI2gSNZJF9FFN__aCYPSk1l_j0WAoe5xly1Oje5Fo/edit?tab=t.0) & [FR](https://docs.google.com/document/d/1nbUtOPg9gAp_vkXO0N4zkWItp148UsAWA-akEyP1Sx8/edit?tab=t.0#heading=h.gjdgxs). 
<img width="719" alt="Screenshot 2024-11-19 at 11 58 55 AM" src="https://github.com/user-attachments/assets/4a2aca5c-8653-4bef-a6f1-b4bac575aee9">

Reply:
I've been wondering is we should repeat "basic page" and "basic page template' in each of the headings like we do for the component pages. These pages tend to be shorter but if you had a few template pages open, could get confusing.

Reply:
I swear I updated that coding heading before my last push last week :D I'll check what happened there and will add it back in!

Reply:
Oh, I remember what happened now—I was missing a few French translations, and because of that, I didn't push the English change either. I'm hoping to get those today, and I will push the changes!